package.json

```diff
@@ -51,52 +51,53 @@
     "postversion": "git push && git push --tags"
   },
   "peerDependencies": {
-    "@angular/core": ">=4.0.0 <5.0.0",
-    "@angular/forms": ">=4.0.0 <5.0.0",
-    "@angular/http": ">=4.0.0 <5.0.0",
+    "@angular/core": ">=4.0.0 <6.0.0",
```

Comment:
are we really still compatible with Angular 4?
At least the syntax of ngc has changed, though that only gets executed at compile-time, I guess…